resources_pages/check-setup-mds.sh

```diff
@@ -286,12 +292,12 @@ if ! [ -x "$(command -v R)" ]; then  # Check that R exists as an executable prog
 else
     # Create an empty Rmd-file for testing
     touch mds-knit-pdf-test.Rmd
-    if ! Rscript -e "rmarkdown::find_pandoc(dir = c('/usr/bin/rstudio/bin/quarto/bin/tools', 'C:/Program Files/RStudio/bin/quarto/bin/tools', '/Applications/RStudio.app/Contents/MacOS/quarto/bin/tools'), cache=F); rmarkdown::render('mds-knit-pdf-test.Rmd', output_format = 'pdf_document')" &> /dev/null; then
+    if ! Rscript -e "rmarkdown::find_pandoc(dir = c('/usr/lib/rstudio/resources/app/bin/quarto/bin/tools', 'C:/Program Files/RStudio/bin/quarto/bin/tools', '/Applications/RStudio.app/Contents/MacOS/quarto/bin/tools'), cache=F); rmarkdown::render('mds-knit-pdf-test.Rmd', output_format = 'pdf_document')" &> /dev/null; then
```

Comment:
My pandoc on my mac is here: `"/Applications/RStudio.app/Contents/Resources/app/quarto/bin/tools"` so we might want to edit this to be:
```suggestion
    if ! Rscript -e "rmarkdown::find_pandoc(dir = c('/usr/lib/rstudio/resources/app/bin/quarto/bin/tools', 'C:/Program Files/RStudio/bin/quarto/bin/tools', '/Applications/RStudio.app/Contents/MacOS/quarto/bin/tools', '/Applications/RStudio.app/Contents/Resources/app/quarto/bin/tools'), cache=F); rmarkdown::render('mds-knit-pdf-test.Rmd', output_format = 'pdf_document')" &> /dev/null; then
```

Reply:
That looks similar to the change in Linux path so I'm guessing RStudio just changed it everywhere since last year. It's probably safe to overwrite the old Mac OS path, Feel free to do that instead and accept your change.

Reply:
I'm worried it's an M1 vs M2 thing, and so I don't feel confident removing last years line. Thus I am opting to keep both for mac this year.